script/install.sh

```diff
@@ -150,6 +150,7 @@ Commands
   install       Install a D compiler (default command)
   uninstall     Remove an installed D compiler
   list          List all installed D compilers
+  where         Finds the binary for a D compiler (installs it if no compiler is found)
```

Comment:
That seems an odd conflation of concerns.  Asking _where_ it is shouldn't auto-install it!

Reply:
What motivation is there behind this assumption that `where` should install if not found?